autoload/neomake/makers/ft/typescript.vim

```diff
@@ -10,6 +10,7 @@ function! neomake#makers#ft#typescript#tsc() abort
         \ 'args': ['--project', neomake#utils#FindGlobFile('tsconfig.json'), '--noEmit', '--watch', 'false'],
         \ 'append_file': 0,
         \ 'errorformat':
+            \ '%E%f(%l\,%c): error %m,' .
             \ '%E%f %#(%l\,%c): error %m,' .
```

Comment:
I don't know -- I copied this from #1204. I never experienced issues with the tsc checker, just the tslint checker.